src/lib/rules/manifest-app-name/manifest-app-name.ts

```diff
@@ -0,0 +1,131 @@
+/**
+ * @fileoverview Check if the web app manifest file includes the `name`
+ * and `short_name` member with appropriate values.
+ */
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const { ucs2 } = require('punycode');
```

Comment:
`punycode` [is deprecated](https://nodejs.org/dist/latest-v8.x/docs/api/punycode.html)
We should use [Punycode.js](https://github.com/bestiejs/punycode.js)

Reply:
[I'm using Punycode.js](https://github.com/alrra/sonar/blob/c6a5c54fec29410934db53968d1fdfa43d22ad3f/package.json#L110), just that it should be a dependency not a devDependency.

Reply:
For whatever reason I though we weren't using it before, I didn't see an update in `package.json` and I assumed the worst :P

Reply:
> For whatever reason I though we weren't using it before,

We used to use it in a rule, but now it's only used in making sure we can use the [correct amount of emoji in commit messages](https://github.com/sonarwhal/sonar/blob/815dd24072d14ff286031dfe089fd358c676c65d/scripts/check-commit-message.js#L3).

> just that it should be a dependency not a devDependency.

Fixed.